internal/explorer/server.go

```diff
@@ -172,6 +172,23 @@ func (a *App) getNode(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(nodeData))
 }
 
+func (a *App) getNodeStatus(w http.ResponseWriter, r *http.Request) {
+	enableCors(&w)
+
+	nodeID := mux.Vars(r)["node_id"]
+	_, err := a.getNodeData(nodeID, false)
+	if err != nil {
+		// return internal server error
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
The errors which happened because the node is down should be handled differently than the errors because of other internal errors (like rmb being down for example). The response status in case it's down should be OK, and in other cases should be internal server error with a response like `{"error": "couldn't send the request because redis is down"}` or something.